backend/lib/geocoder.py

```diff
@@ -103,7 +112,9 @@ def geocode(self, address):
                 'latitude': result[0],
                 'longitude': result[1],
             }
-        except:
+        # Note - Ox raises general exceptions Exception.
+        except Exception as error:
+            logger.exception(error)
```

Comment:
same thing here. do we want to logger exception, or actually raise the exception, or just logger error?

Reply:
Done